src/SFA.DAS.EAS.Employer_Financial.Database/StoredProcedures/GetAllPaymentsForLastYearByAccountId.sql

```diff
@@ -0,0 +1,8 @@
+﻿CREATE PROCEDURE [employer_financial].[GetAllPaymentsForLastYearByAccountId]
+	@AccountId BIGINT
+AS
+	SELECT SUM(Amount)*-1 AS TotalSpend 
```

Comment:
Should we use the ABS() SQL function to get the positive number from the negative payment value?

Reply:
No because we need to account for the super uber rare scenario when there is more refunds then payments